plugins/event_notification/providers/email/lib/batch/recipients/KEMailNotificationRecipientEngine.php

```diff
@@ -36,4 +36,57 @@ public static function getEmailNotificationRecipientEngine(KalturaEmailNotificat
 	 * @param array $contentParameters
 	 */
 	abstract function getRecipients (array $contentParameters);
+
+	protected function getUsersOfGroupByGroupId($groupId)
+	{
+		$groupFilter = new KalturaGroupUserFilter();
+		$groupFilter->groupIdEqual = $groupId;
+		$pager = new KalturaFilterPager();
+		$pager->pageSize = 500;
+		$userFilter = new KalturaUserFilter();
+
+		$groupUserList = KBatchBase::$kClient->groupUser->listAction($groupFilter, $pager);
+		if(!($groupUserList->totalCount))
+		{
+			return null;
+		}
+
+		$groupUserIds = array();
+		foreach ($groupUserList->objects as $user)
+		{
+			$groupUserIds[]= $user->userId;
+		}
+
+		$maxIdInQueryChunk = 150;
+		$pager->pageSize = $maxIdInQueryChunk;
+		$offset = 0;
+		$allUsers = array();
+
+		while($offset < count($groupUserIds))
+		{
+			$currentUserArrIds = array_slice($groupUserIds, $offset, $maxIdInQueryChunk);
+			$currentUserStrIds = implode(',', $currentUserArrIds);
+			$userFilter->idIn = $currentUserStrIds;
+			$response = KBatchBase::$kClient->user->listAction($userFilter, $pager);
+			if($response)
```

Comment:
lets do the oposite:
if(!$response)
   print and break

no need for else
on lines 81 and 82 just use 
$response->objects and $response->totalCount; without spaces before and after ->